smart_control/simulator/stochastic_occupancy.py

```diff
@@ -49,217 +48,239 @@
 
 
 class OccupancyStateEnum(enum.Enum):
-    AWAY = 1
-    WORK = 2
+  AWAY = 1
+  WORK = 2
+
 
 class ZoneOccupant:
-    def __init__(
-        self,
-        earliest_expected_arrival_hour: int,
-        latest_expected_arrival_hour: int,
-        earliest_expected_departure_hour: int,
-        latest_expected_departure_hour: int,
-        lunch_start_hour: int,
-        lunch_end_hour: int,
-        step_size: pd.Timedelta,
-        random_state: np.random.RandomState,
-        time_zone: Union[datetime.tzinfo, str] = 'UTC',
+
+  def __init__(
+      self,
+      earliest_expected_arrival_hour: int,
+      latest_expected_arrival_hour: int,
+      earliest_expected_departure_hour: int,
+      latest_expected_departure_hour: int,
+      lunch_start_hour: int,
+      lunch_end_hour: int,
+      step_size: pd.Timedelta,
+      random_state: np.random.RandomState,
+      time_zone: Union[datetime.tzinfo, str] = "UTC",
+  ):
+    assert (
```

Comment:
let's remove these assertions from non-test code. 

> "Do not use assert statements in place of conditionals or validating preconditions." - [source](https://google.github.io/styleguide/pyguide.html#24-exceptions)

we can move this to a separate issue